(window.webpackJsonp=window.webpackJsonp||[]).push([[27],{406:function(e,t,a){"use strict";a.r(t);var n=a(19),r=Object(n.a)({},(function(){var e=this,t=e.$createElement,a=e._self._c||t;return a("ContentSlotsDistributor",{attrs:{"slot-key":e.$parent.slotKey}},[a("h1",{attrs:{id:"scatac-v1-2-0-methods"}},[a("a",{staticClass:"header-anchor",attrs:{href:"#scatac-v1-2-0-methods"}},[e._v("#")]),e._v(" scATAC v1.2.0 Methods")]),e._v(" "),a("p",[e._v("Below we provide a sample methods section for a publication. For the complete pipeline documentation, see the "),a("RouterLink",{attrs:{to:"/documentation/Pipelines/Single_Cell_ATAC_Seq_Pipeline/"}},[e._v("scATAC Overview")]),e._v(".")],1),e._v(" "),a("h2",{attrs:{id:"methods"}},[a("a",{staticClass:"header-anchor",attrs:{href:"#methods"}},[e._v("#")]),e._v(" Methods")]),e._v(" "),a("p",[e._v("Preprocessing and cell by bin matrix construction were performed using the scATAC v1.2.0 Pipeline.")]),e._v(" "),a("p",[e._v("Prior to processing, paired-end FASTQ files were modified with a custom python script (available at https://github.com/r3fang/SnapTools/blob/master/snaptools/dex_fastq.py) so that readnames were appended with cell barcodes.")]),e._v(" "),a("p",[e._v("The appended reads were then aligned to the Hg38 genomic reference using BWA v0.7.17. The resulting aligned BAM was converted into fragments and filtered using the "),a("a",{attrs:{href:"https://github.com/r3fang/SnapTools",target:"_blank",rel:"noopener noreferrer"}},[e._v("SnapTools v1.4.7"),a("OutboundLink")],1),e._v(" SnapPre function with default parameters.")]),e._v(" "),a("p",[e._v("The snap-add-bmat function was then used to add cell-by-bin matrices to the resulting Snap file. 10 kb was selected as the default value for the bin size based on the SnapTools recommendation for mid-size datasets. It can be changed by specifying the desired size as an input to this workflow.")]),e._v(" "),a("p",[e._v("Custom python scripts were then used to make a "),a("a",{attrs:{href:"https://github.com/broadinstitute/warp/blob/master/dockers/skylab/snaptools/makeCompliantBAM.py",target:"_blank",rel:"noopener noreferrer"}},[e._v("GA4GH-compliant BAM"),a("OutboundLink")],1),e._v(" and to export select Snap file metrics to "),a("a",{attrs:{href:"https://github.com/broadinstitute/warp/blob/master/dockers/skylab/snap-breakout/breakoutSnap.py",target:"_blank",rel:"noopener noreferrer"}},[e._v("individual text files"),a("OutboundLink")],1),e._v(".")]),e._v(" "),a("p",[e._v("An example of the pipeline and its outputs is available on "),a("a",{attrs:{href:"https://app.terra.bio/#workspaces/brain-initiative-bcdc/SnapATAC_Pipeline",target:"_blank",rel:"noopener noreferrer"}},[e._v("Terra"),a("OutboundLink")],1),e._v(" and more documentation can be found at "),a("RouterLink",{attrs:{to:"/documentation/Pipelines/Single_Cell_ATAC_Seq_Pipeline/"}},[e._v("here")]),e._v(".\nExamples of genomic reference files and other inputs can be found in the pipelineâ€™s "),a("a",{attrs:{href:"https://github.com/broadinstitute/warp/blob/master/pipelines/skylab/scATAC/example_inputs/human_example.json",target:"_blank",rel:"noopener noreferrer"}},[e._v("example JSON"),a("OutboundLink")],1),e._v(".")],1)])}),[],!1,null,null,null);t.default=r.exports}}]);